charts/argocd-applicationset/ci/servicemonitor-values.yaml

```diff
@@ -0,0 +1,8 @@
+metrics:
+  enabled: true
+  serviceMonitor:
+    enabled: true
+    # -- Prometheus ServiceMonitor interval
+    interval: 30s
+    additionalLabels:
+      release: kube-prometheus-stack
```

Comment:
I think this does not make sense here since our CI (github action) has no Prometheus/Prometheus CRDs installed.

Reply:
Yes, it was used in my own test cluster. Just removed this.

Reply:
You have to drop the part `serviceMonitor.enabled: true`. We neither have Prometheus nor the CRDs in place during CI.

Reply:
Sorry about that. I'm not familiar with GitHub actions and just wondering if we can test some values just against lint/template since some dependencies are not available when test installation.